Sources/DeepLearning/Layer.swift

```diff
@@ -1292,8 +1292,8 @@ public extension RNNCell {
 
 /// A simple RNN cell.
 public struct SimpleRNNCell<Scalar: TensorFlowFloatingPoint>: RNNCell, VectorNumeric {
-    public var weight: Tensor<Scalar>
-    public var bias: Tensor<Scalar>
+    public var W_h: Tensor<Scalar>
+    public var b_h: Tensor<Scalar>
 
     @noDerivative public var stateShape: TensorShape {
         return TensorShape([1, weight.shape[1]])
```

Comment:
I think this line doesn't compile, and should be changed to:

```suggestion
        return TensorShape([1, W_h.shape[1]])
```